zerver/webhooks/delighted/view.py

```diff
@@ -37,12 +38,12 @@ def body_template(score: int) -> str:
 def api_delighted_webhook(
     request: HttpRequest,
     user_profile: UserProfile,
-    payload: Dict[str, Dict[str, Any]] = REQ(argument_type="body"),
+    payload: WildValue = REQ(argument_type="body", converter=to_wild_value),
 ) -> HttpResponse:
     person = payload["event_data"]["person"]
-    selected_payload = {"email": person["email"]}
-    selected_payload["score"] = payload["event_data"]["score"]
-    selected_payload["comment"] = payload["event_data"]["comment"]
+    selected_payload: Dict[str, Any] = {"email": person["email"].tame(check_string)}
+    selected_payload["score"] = payload["event_data"]["score"].tame(check_int)
+    selected_payload["comment"] = payload["event_data"]["comment"].tame(check_string)
 
     BODY_TEMPLATE = body_template(selected_payload["score"])
     body = BODY_TEMPLATE.format(**selected_payload)
```

Comment:
During this migration we should be removing all use of `Any` in these files ([explanation](https://zulip.readthedocs.io/en/latest/testing/mypy.html#avoid-the-any-type)), so that we can be confident that the migration isn’t introducing bugs hidden behind unchecked conversions from `Any`.

This will mean a bit of extra rewriting in some cases. For example,

```suggestion
    email = person["email"].tame(check_string)
    score = payload["event_data"]["score"].tame(check_int)
    comment = payload["event_data"]["comment"].tame(check_string)

    BODY_TEMPLATE = body_template(score)
    body = BODY_TEMPLATE.format(email=email, score=score, comment=comment)
```

which is a readability improvement anyway.